app/controllers/taxons_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TaxonsController < ApplicationController
+  before_action :redirect_to_www, unless: :new_navigaton_enabled?
+
+  def show
+    render :show, locals: {
```

Comment:
Ok, we can reduce the number of locals being passed.

I am in general against instance variables being set in controllers and then used in views. It's better to explicitly pass variables need so we 1) detect errors when we don't 2) avoid having partials inside partials relying on those instance variables.

If you agree with this, I think we can make an effort to migrate things as we touch them and also review PRs taking this into account. What do you think?